gyp/.flake8

```diff
@@ -1,4 +1,4 @@
 [flake8]
-max-complexity = 10
+max-complexity = 101
```

Comment:
Ouch!

Reply:
heh, take it up with gyp-next, I'm just vendoring it in